.github/workflows/build.yml

```diff
@@ -21,12 +21,11 @@ jobs:
           - windows-latest
 
         php:
-          - "7.4"
           - "8.0"
 
     steps:
       - name: Checkout
-        uses: actions/checkout@v2
+        uses: actions/checkout@v2.3.4
```

Comment:
It was alright.

Reply:
Also need fix in package template?

https://github.com/yiisoft/package-template/blob/85a5a35d6615ab116b3df9362f64d7fca96c3fae/.github/workflows/build.yml#L28

Reply:
Yes.

Reply:
Done.